src/test/java/com/training/services/tests/GACreateTest.java

```diff
@@ -54,10 +48,22 @@ public void  testGuestCreation(){
         requestBodyCreate.setAcceptDateTime("20190524T090712GMT");
         requestBodyCreate.setVersion("1.8");
         requestBodyCreate.setUidType("EMAIL");
-        Response responseGACreate  =
+
+    }
+
+    public void getResponse(){
```

Comment:
Make this method more specific - this should return Response, and should take in the requestBody as parameter